man/mustache.5.ron

```diff
@@ -411,6 +411,37 @@ Can be thought of as a single, expanded template:
     {{/names}}
 
 
+**Dynamic Names**
+
+Partials can be loaded dynamically at runtime using Dynamic Names; an
+**optional** special notation which allows to dynamically determine a tag's
+content at runtime.
+
+Dynamic Names consists of an asterisk, followed by a dotted name which follows
+the same notation and the same resolution as in an Interpolation tag. That is
+`{{>*dynamic}}`.
+
+Failed resolution of the dynamic name SHOULD result in nothing being rendered.
```

Comment:
I think this sentence can be left out. The manpage doesn't spend much attention to this corner case for most of the other tags, either.